ui/app/views/deployments/new.html

```diff
@@ -254,7 +254,8 @@ <h4 class="modal-title" id="myModalLabel">Fashizle?</h4>
                         </div>
                         <div class="modal-footer">
                             <button type="button" class="btn btn-default" data-dismiss="modal">Close</button>
-                            <button ng-disabled="environmentSelected.requireDeploymentMessage && deploymentMessage.text === undefined" type="button" class="btn btn-primary" data-dismiss="modal" ng-click="deploy()">Just deploy already!</button>
+                            <button ng-disabled="environmentSelected.requireDeploymentMessage && deploymentMessage.text === undefined" type="button" class="btn btn-primary" data-dismiss="modal" ng-click="deploy(0)">Just deploy already!</button>
+                            <button ng-disabled="environmentSelected.requireDeploymentMessage && deploymentMessage.text === undefined" type="button" class="btn btn-primary" data-dismiss="modal" ng-click="deploy(1)">Emergency Rollback</button></div>
```

Comment:
I want to show another modal here that  explains the risks of doing that. Work  with me on  the text

Reply:
Let's talk :)